cumulus/etc/iproute2/rt_protos.d/frr.conf

```diff
@@ -0,0 +1,8 @@
+# Additional protocol strings defined by quagga for each of its daemons
```

Comment:
Yes agreed, Per our slack conversation -> we will put this into master with appropriate checks in place.